recipes/uchardet/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class UchardetConan(ConanFile):
+    name = "uchardet"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/freedesktop/uchardet"
+    description = "uchardet is an encoding detector library, which takes a sequence of bytes in an unknown character encoding and attempts to determine the encoding of the text. Returned encoding names are iconv-compatible."
+    topics = "encoding", "detector"
+    license = "MPL 1.1"
```

Comment:
https://spdx.org/licenses/MPL-1.1.html

```suggestion
    license = "MPL-1.1"
```